packages/testing-library/utils/adder.ts

```diff
@@ -12,7 +12,11 @@ export async function runAdder(
     adderOptions[adder.config.metadata.id] = optionValues;
     const remoteControlOptions: RemoteControlOptions = { workingDirectory, isTesting: true, adderOptions };
 
-    await executeAdder({ config: adder.config, checks: adder.checks }, remoteControlOptions);
+    await executeAdder(
+        { config: adder.config, checks: adder.checks },
+        { name: "testing-cli", version: "tbd" },
```

Comment:
is this a leftover?

Reply:
This generally needs to be there, but I have adjusted the naming to better reflect the package name